design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -257,6 +258,48 @@ VMI status and in one case to pass the data (processed) to the
 
 - DownwardAPI
 
+##### Device Info
+
+In some cases the multus status annotation
+contains [device-info](https://github.com/k8snetworkplumbingwg/device-info-spec/blob/main/SPEC.md).
+This info may be valuable for the binding plugin sidecar. For example
+`SRIOV` and `VDPA` plugins need this info.
+
+As the multus status annotation is not there yet on `virt-launcher` pod
+creation, the suggested solution is to use downwardAPI volume and mount
+it to the sidecar.
+The downwardAPI volume will track a new `virt-launcher` pod
+annotation `kubevirt.io/network-device-info-map` that will contain a
```

Comment:
Please consider removing map prefix, IMO it doesn't add value.
Also if tomorrow we decide to change the annotation value format (to not be map) the name would no longer be valid.

Reply:
The fact it is a map is part of the API. I cannot change it to be something else without breaking backward compatibility.

Reply:
I do not think the meaning of `map` is universally understood.
In the `Go` programming language a key-value structure is named `map`, in other languages it is called a `dict` (Python).
In plain English it describes an are of land/sea.

Even in programming we try to avoid expressing the type of a variable in the name. We usually prefer the name to represent what the data contains and not how it contains it.

I also think the `map` is redundant.